functions/.runtimeconfig.json.example

```diff
@@ -2,6 +2,9 @@
   "project": {
     "id": ""
   },
+  "auth": {
+    "origin": "https://foo.example.com"
+  },
```

Comment:
[TODO] Do we want to keep this here, or move it into the `authConfig` in firestore? I think `authConfig`. And maybe name it better. It's essentially used as part of the 'redirect back to the platform' code.

Reply:
`authConfig` won't work well though, as it will need to be different between both hosted `staging` and local `staging`. So I think it might need to stay as it is here.

Reply:
Though if we make it a list of whitelisted origins in `authConfig`, then we could read it from the headers/url params of the request to the function, check that it's on the whitelist, and if it's valid we can use it.

That way we can configure it to work in multiple environments, and we avoid the security issues of blindly trusting the headers/URL params.